src_c/time.c

```diff
@@ -263,13 +257,7 @@ static PyObject *
 time_wait(PyObject *self, PyObject *arg)
 {
     int ticks, start;
-    PyObject *arg0;
-
-    /*for some reason PyArg_ParseTuple is puking on -1's! BLARG!*/
-    if (PyTuple_Size(arg) != 1)
-        return RAISE(PyExc_ValueError, "delay requires one integer argument");
-    arg0 = PyTuple_GET_ITEM(arg, 0);
-    if (!PyLong_Check(arg0))
+    if (!PyLong_Check(arg))
         return RAISE(PyExc_TypeError, "delay requires one integer argument");
```

Comment:
Not your fault, but it looks like someone made a copy paste error here. Shouldn't it be "wait requires one integer argument" ?

Reply:
good catch